README.md

```diff
@@ -59,7 +59,9 @@ the 4th byte in the stream.
 
 ##### etag
 
-Enable or disable etag generation, defaults to true.
+Accepts `'weak'`, `'strong'`, custom function `function(data){ return etag}`,
```

Comment:
It is not clear what the "data" argument is in the documentation. Please elaborate in the document what this is and what it contains.